status-im/status-react
# fix #1518 removed potentially offensive words from generated names

### Summary:
Removed any potentially offensive words from the lists used to automatically generate new account name, e.g:

- completely inappropriate words such as ass and cock (even though their original meaning was quite innocent - donkey and rooster, resp.) 
- offensive adjectives, such as idiotic, dumb
- adjectives that describe a negative sentiment, e.g. depressive, sad
- adjectives that imply race or gender, e.g. black, feminine, fatherly.

### Steps to test:

- Open app
- Create a new user
- The generated name should not contain any of the controversial words

status: ready
